.appveyor.yml

```diff
@@ -9,8 +9,8 @@ cache:
 
 environment:
     global:
-        CONDA_CHANNELS: conda-forge
-        CONDA_DEPENDENCIES: pip setuptools wheel cython biopython networkx joblib matplotlib scipy vs2015_runtime pytest mmtf-python GridDataFormats hypothesis pytest-cov codecov chemfiles tqdm tidynamics>=1.0.0 rdkit>=2020.03.1 h5py
+        CONDA_CHANNELS: conda-forge omnia
```

Comment:
Just a thought for the future, we should raise an issue here to remind us to keep track of when openmm goes full conda-forge (I know it's in beta right now).